### PR TITLE
Created <Sponsors/> Component

### PR DESCRIPTION
**Pull request by adityarai0705**
Use this component by,
        <Sponsors ImageList={List} />

where List is an Array of size 32 that contains a relative path to the images of sponsors in text format.

IMPORTANT: Changing the size of the Array ( the number of sponsors ) will require some changes in the code too.

for example,

List = [
    ...
    './SponsorsImages/ecellLogo.png',
    ...
]

### CODE REVIEW

**Review by Ashish-Verma-MNNIT — CHANGES_REQUESTED** [review]
There are certain rules written in .eslintrc. your code is not consistent with it thus giving many errors like...
image
Please correct it.

File: client/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "editor.codeActionsOnSave": { "source.fixAll": true },
+    "editor.formatOnSave": false,
```
Comment (sanskaromar) [inline client/.vscode/settings.json:3]: This should be true. Why false?

File: client/src/components/Sponsors/READ ME.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (sanskaromar) [inline client/src/components/Sponsors/READ ME.txt:1]: rename the file to README.md instead of READ ME.md

**Review by sanskaromar — CHANGES_REQUESTED** [review]
Why did you push .DS_Store ? Add it to .gitignore if you want, or just don't commit it. For now, remove it.

File: client/src/App.js [hunk 3]
```diff
@@ -1,25 +1,8 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return <></>;
```
Comment (PriyavKaneria) [inline client/src/App.js:5]: You should atleast leave a working sample usage in this file. Readme is not clear enough

**Review by PriyavKaneria — CHANGES_REQUESTED** [review]
image
Unable to use the component even after using it as per mentioned in readme
Either add sample usage or fix the issues. We won't be editing the code to make it run just to review it

File: client/src/App.js (hunk 3 above)
Comment (adityarai0705) [reply client/src/App.js]: I have updated the PR with a working sample of the component in App.js. Please check it.

File: client/.vscode/settings.json (hunk 1 above)
Comment (adityarai0705) [reply client/.vscode/settings.json:3]: Okay sir, I changed it to true.

File: client/src/components/Sponsors/READ ME.txt (hunk 2 above)
Comment (adityarai0705) [reply client/src/components/Sponsors/READ ME.txt:1]: Changed this too.

**Review by PriyavKaneria — DISMISSED** [review]
Shift the wheel section a little lower and make everything non selectable
https://stackoverflow.com/a/4407335
Nice work

**Comment by NeerajChatterjee** [comment]
image
- The size of the sponsor card should be changed in mobile view, because the gradient and size don't fit at that size.
Other than that, nice work.

**Review by PriyavKaneria — APPROVED** [review]
Perfect

**Comment by adityarai0705** [comment]
I have reduced the SponsorsCard size by 25% for displays with width less than 480px.

**Comment by adityarai0705** [comment]
> image
> 
> * The size of the sponsor card should be changed in mobile view, because the gradient and size don't fit at that size.
>   Other than that, nice work.

Reduced SponsorsCard size for mobile devices

**Comment by adityarai0705** [comment]
> There are certain rules written in .eslintrc. your code is not consistent with it thus giving many errors like...
> image
> Please correct it.

I have made changes as you asked. Please check it.

**Comment by NeerajChatterjee** [comment]
> > image
> > 
> > * The size of the sponsor card should be changed in mobile view, because the gradient and size don't fit at that size.
> >   Other than that, nice work.
> 
> Reduced SponsorsCard size for mobile devices

Okay fine, we will merge your PR shorty.

**Pull request merged**